### DISCUSSION TITLE
how to give commit access to other user

### DISCUSSION THREAD

**Discussion by navindigicita** [vercel/community#7316] [Help]
### Summary

Hey there,

My git project is connected to Vercel, but git repo owner isn't able to deploy on the vercel but one of collaborator of the same git repo(who is another user) is easily able to commit and deploy the code on vercel. I want to add git repo owner as a commit access person. Tried sharing invite for access, in fact changed git user name as well in project authentication. 

Please help me in this.

### Steps to Reproduce

setting -> git -> added git repo

**Comment by pawlean** [comment]
Hi @navindigicita!

So that I understand and can help you out here who is the owner of the Git project?

**Reply by pawlean** [reply]
This seems like an issue with permissions regardless, these are some things to try:

1. Ensure that the repository owner has proper permissions set on GitHub (or the respective Git service) to commit and push changes.
2. Check Vercel's team settings to confirm that the owner is added as a member with the necessary permissions.
3. Reconnect the GitHub account on Vercel for the repository owner to refresh any token or authentication issues.
4. Verify the project's settings on Vercel to ensure the repository owner is listed as a collaborator with deployment permissions.
5. Make sure any required access tokens or SSH keys are correctly configured for the repository owner.

[a comment by an automated account is not shown]
